plugins/woocommerce/src/Admin/Loader.php

```diff
@@ -55,4 +70,20 @@ public static function is_embed_page() {
 		wc_deprecated_function( 'is_embed_page', '6.3', '\Automattic\WooCommerce\Admin\PageController::is_embed_page()' );
 		return PageController::is_embed_page();
 	}
+
+	/**
+	 * Determines if a minified JS file should be served.
+	 *
+	 * @param  boolean $script_debug Only serve unminified files if script debug is on.
+	 * @return boolean If js asset should use minified version.
+	 *
+	 * @deprecated since 6.3.0, use WCAdminAssets::should_use_minified_js_file( $script_debug )
+	 */
+	public static function should_use_minified_js_file( $script_debug ) {
+		// Bail if WC isn't initialized (This can be called from WCAdmin's entrypoint).
+		if ( ! defined( 'WC_ABSPATH' ) ) {
+			return;
+		}
+		return WCAdminAssets::should_use_minified_js_file( $script_debug );
```

Comment:
No deprecation notice via `wc_deprecated_function` since it also causes output during activation, which would break plugin activation.